account_invoice_line_sequence/models/invoice.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    sequence = fields.Integer(help="Gives the sequence of this line when "
```

Comment:
Could you add a comment in the code about this re-definition?
(Because we have this same field from [9.0 - account_invoice.py](https://github.com/odoo/odoo/blob/26627391870af2edd8315748929122392a9a5f68/addons/account/models/account_invoice.py#L1129-L1130))